### DISCUSSION TITLE
Is It possible to build OpenDDS 3.26.1 on Windows 10 as Win32?

### DISCUSSION THREAD

**Discussion by kubraturhan** [OpenDDS/OpenDDS#4464] [Q&A]
I could not build DDS_no_tests.sln file as Release Win32. Is it only way to build as Win32 using Windows 7 machine?

**Comment by mitza-oci** [comment]
Please describe the steps you took and the error that you saw.  I don't know of any difference among Windows 7, 8.x, 10, 11 as far as OpenDDS is concerned.
